shell/cmd/cosh/test.cosh

```diff
@@ -1,6 +1,7 @@
 // test file for cosh cli
 
-echo {args}
+// todo: no longer works
+// echo {args}
```

Comment:
What about fixing this?

Reply:
I forgot what we did to args -- working on other things now but if you remember, could you perhaps just do it?

Reply:
We seem to handle it in the `Run` command, so I don't think it is necessarily supposed to work with `Build`:

```
	if len(c.Args) > 0 {
		in.Eval("args := cosh.StringsToAnys(" + fmt.Sprintf("%#v)", c.Args))
	}
```